core.impl/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/impl/locks/CASLockFactoryBuilder.java

```diff
@@ -37,6 +38,7 @@ public class CASLockFactoryBuilder
     private long myCacheExpiryTimeInSeconds = DEFAULT_EXPIRY_TIME_IN_SECONDS;
     private ConsistencyType myConsistencyType = DEFAULT_CONSISTENCY_SERIAL;
     private Node myNode;
```

Comment:
It should be a method parameter

Reply:
did not get it. which method parameter?? we use uuid while building CASLockFactory object in constructor. This is not exposed directly in public method. I am going to remove Node and replacing with uuid because we do not need Node object. UUID is enough.